### PR TITLE
Add normalize.scss as dependency

### PR DESCRIPTION
**Pull request by misscs**
This PR adds normalize.scss as dependency and imports the partial at the top of `base_styles.scss`.

### CODE REVIEW

File: src/_base_styles.scss [hunk 1]
```diff
@@ -3,6 +3,9 @@
  * @overview Site scaffolding and helper classes that generate output
  */
 
+// Third party
+@import './node_modules/normalize.scss/normalize';
+
 @import 'typography/index';
```
Comment (robwierzbowski) [inline src/_base_styles.scss:8]: Can you add a header on line 9 too, to avoid confusion?

File: src/_base_styles.scss [hunk 2]
```diff
@@ -3,6 +3,9 @@
  * @overview Site scaffolding and helper classes that generate output
  */
 
+// Third party
```
Comment (victorwhy) [inline src/_base_styles.scss]: i think a little more description than `Third Party` would be great

**Comment by victorwhy** [comment]
looks good! i think just a couple more updates to comments and I'm good with it!

**Comment by misscs** [comment]
@robwierzbowski and @victorwhy update comment in the file. let me know what you think https://github.com/CasperSleep/nightshade-core/pull/147/commits/c50c215f280169ba43f35c568b9814df92a93a00

**Comment by robwierzbowski** [comment]
DTM!

**Comment by misscs** [comment]
Thank you!

**Pull request merged**